challenges/prototypes.js

```diff
@@ -6,28 +6,48 @@
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
 
+function CuboidMaker(obj) {
+  this.length = obj.length;
+  this.width = obj.width;
+  this.height = obj.height;
+}
+
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
   
   Formula for cuboid volume: length * width * height
 */
 
+CuboidMaker.prototype.volume = function () {
+  return this.length * this.width * this.height
+}
+
 
 /* == Step 3: Surface Area Method ==
   Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height. 
 
   Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
 */
 
+CuboidMaker.prototype.surfaceArea = function () {
+  return this.length * this.width + this.length * this.height + this.width + this.height
+}
+
```

Comment:
just a minor thing, the formula is  **2 *** (length * width + length * height + width * height)